docs/references/javascript/types/overview.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Clerk types
+description: Explore the different types available for typing your application.
+---
+
+# Clerk types
+
+Types are a powerful tool for adding type-safety to your application. They can help you catch bugs early, make your code easier to understand, and make your code easier to refactor. Clerk provides a number of types to help you add type-safety to your application.
+
+To get access to Clerk types, you need to add the `@clerk/types` package to your project. Install it by running the following command in your terminal:
```

Comment:
Arent we getting rid of the types package soon in favor of the types being directly available in the SDK's? 

Cc: @nikosdouvlis @panteliselef